### PR TITLE
cip-190 deprecations

### PR DESCRIPTION
**Pull request by hvub**
Deprecation message related to

- https://github.com/neo-technology/neo4j/pull/27564
- https://github.com/neo4j/docs-cypher/pull/1084
- cf. [CIP-190](https://docs.google.com/document/d/1AONjI-SpSEla-bHPJ6ghXBfF9alN5mGiOHOTEyfBtRc/edit)

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-status-codes-202.surge.sh

Updated pages:
- [List of all server notification codes](https://neo4j-docs-status-codes-202.surge.sh/status-codes/5/notifications/all-notifications)

File: modules/ROOT/pages/notifications/all-notifications.adoc [hunk 1]
```diff
@@ -2183,9 +2320,39 @@ END AS standardsName
 ======
 =====
 
-.Using an unescaped variable named `contains` in addition or subtraction operations within a `WHEN` operand in a simple `CASE` expression
+.Using an unescaped variable named `contains` for relationships in addition or subtraction operations within a `WHEN` operand in a simple `CASE` expression
```
Comment (hvub) [inline modules/ROOT/pages/notifications/all-notifications.adoc:2323]: The variable `contains` in the example bind to an INTEGER (the number of relationships in the path) and that is primarily a characteristics of the specific examples. 

As more abstract example would be:
```
WITH 2 AS x, 1 AS contains
RETURN CASE x
  WHEN contains + 1 THEN "foo"
  ELSE "bar"
END AS check
```
This does not involve any nodes and relationships — to make the point that nodes and relationships are not integral to this deprecations.

File: modules/ROOT/pages/notifications/all-notifications.adoc [hunk 2]
```diff
@@ -2021,6 +2021,584 @@ warn: feature deprecated without replacement.
...
++
+[source,cypher]
+----
+MATCH (`where` {p: 5})
+RETURN `where`.p
+----
+======
+=====
+
+.Using an unescaped variable named 'where' in relationship patterns
```
Comment (hvub) [inline modules/ROOT/pages/notifications/all-notifications.adoc:2080]:
```suggestion
.Using an unescaped variable named 'where' in a relationship pattern
```
For consistency with the above suggestion. Plural vs. singular is your call. It just seems to me singular is used more on this page.

File: modules/ROOT/pages/notifications/all-notifications.adoc [hunk 3]
```diff
@@ -2021,6 +2021,584 @@ warn: feature deprecated without replacement.
 ======
 =====
 
+.Using an unescaped variable named 'where' in node
```
Comment (hvub) [inline modules/ROOT/pages/notifications/all-notifications.adoc:2024]:
```suggestion
.Using an unescaped variable named 'where' in a node pattern
```

File: modules/ROOT/pages/notifications/all-notifications.adoc [hunk 4]
```diff
@@ -2021,6 +2021,584 @@ warn: feature deprecated without replacement.
...
+WITH n, n.internationalStandard AS `is`
+RETURN CASE n
+  WHEN `is` :: INTEGER THEN "ISO/IEC" + `is`
+  ELSE `is`
+END AS standardsName
+----
+======
+=====
+
+.Using an unescaped variable named `contains` for relationships in addition or subtraction operations within a `WHEN` operand in a simple `CASE` expression
```
Comment (hvub) [inline modules/ROOT/pages/notifications/all-notifications.adoc:2323]:
```suggestion
.Using an unescaped variable named `contains` in addition or subtraction operations within a `WHEN` operand in a simple `CASE` expression
```

The variable contains in the example bind to an INTEGER (the number of relationships in the path) and that is primarily a characteristics of the specific examples.

As more abstract example would be:
```
WITH 2 AS x, 1 AS contains
RETURN CASE x
  WHEN contains + 1 THEN "foo"
  ELSE "bar"
END AS check
```
This does not involve any nodes and relationships — to make the point that nodes and relationships are not integral to this deprecations.

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-status-codes-202.surge.sh

Updated pages:
- [List of all server notification codes](https://neo4j-docs-status-codes-202.surge.sh/status-codes/5/notifications/all-notifications)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-status-codes-202.surge.sh

Updated pages:
- [List of all server notification codes](https://neo4j-docs-status-codes-202.surge.sh/status-codes/5/notifications/all-notifications)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-status-codes-202.surge.sh

Updated pages:
- [List of all server notification codes](https://neo4j-docs-status-codes-202.surge.sh/status-codes/5/notifications/all-notifications)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-status-codes-202.surge.sh

Updated pages:
- [List of all server notification codes](https://neo4j-docs-status-codes-202.surge.sh/status-codes/5/notifications/all-notifications)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-status-codes-202.surge.sh

Updated pages:
- [List of all server notification codes](https://neo4j-docs-status-codes-202.surge.sh/status-codes/5/notifications/all-notifications)

**Review by renetapopova — APPROVED** [review]
I think it looks good now. If you agree, I can merge it after we publish 5.25.

**Comment by hvub** [comment]
Looks good. Please merge. Thanks. :-)

**Pull request merged**